compute_endpoint/tests/unit/test_worker.py

```diff
@@ -158,6 +154,9 @@ def sleeper(t):
     return True
 
 
+@pytest.mark.skip(
+    "Timeout is done by the helper.execute_task method and not by the worker"
+)
 def test_app_timeout(test_worker):
```

Comment:
Same here: let's not skip this either, but refactor it to match the new code.  The user -- in this case, most likely the administrator -- will still expect tasks to be limited if this environment variable is set.

Reply:
This test is now fixed.